tools/cldr-apps/src/main/java/org/unicode/cldr/web/UserRegistry.java

```diff
@@ -347,55 +350,19 @@ public int hashCode() {
          * is the user interested in this locale?
          */
         public boolean interestedIn(CLDRLocale locale) {
-            return LocaleNormalizer.localeMatchesLocaleList(intlocs, locale);
+            return getInterestLocales().contains(locale);
         }
 
         /**
-         * List of interest groups the user is interested in.
+         * Set of interest locales for this user
          *
-         * @return list of locales, or null for ALL locales, or a 0-length list
-         *         for NO locales.
+         * @return null or LocaleNormalizer.ALL_LOCALES_SET for 'all', otherwise a set of CLDRLocales
          */
-        @Schema( hidden = true )
-        public String[] getInterestList() {
-            if (userIsManagerOrStronger(this)) {
-                if (intlocs == null || intlocs.length() == 0) {
-                    return null;
-                } else {
-                    if (intlocs.equalsIgnoreCase(LocaleNormalizer.NO_LOCALES)) {
-                        return new String[0];
-                    }
-                    if (LocaleNormalizer.isAllLocales(intlocs)) return null; // all = null
-                    return LocaleNormalizer.tokenizeLocale(intlocs);
-                }
-            } else if (userIsStreet(this)) {
-                if (LocaleNormalizer.isAllLocales(locales)) return null; // all = null
-                return LocaleNormalizer.tokenizeLocale(locales);
-            } else {
-                return new String[0];
-            }
-        }
-
-        /**
-         * Set of interest locales for this user.
-         * @return null for 'all', otherwise a set of CLDRLocales
-         */
-        @Schema( hidden = true ) // TODO: does not seem to effectively hide
-        public Set<CLDRLocale> getInterestLocales(boolean ignoredParameter) {
```

Comment:
!!!! @btangmu I missed this.   

Looks like @Schema was dropped (why?)  and ignoredParameter . They were added to prevent this function from being used in serialization.

Reply:
> Looks like `@Schema` was dropped (why?) and ignoredParameter . They were added to prevent this function from being used in serialization.

@srl Sorry for the misunderstanding. The comment seemed to mean that `@Schema` here was ineffective. `ignoredParameter` seemed to be cruft. Do either of them actually accomplish something useful? If so, let's restore them along with a clarifying comment.

Reply:
Schema in other places is effective,  `ignoredParameter` was to change the serializer.  (This was part of the issue with serializing UserRegistry.User). Actually, you had dropped the serialization of User from Auth, so it is moot for now.  In any event, future modernization.